BedrockServer.cpp

```diff
@@ -1887,9 +1883,9 @@ void BedrockServer::_control(unique_ptr<BedrockCommand>& command) {
         _shouldBackup = true;
         _beginShutdown("Detach", true);
     } else if (SIEquals(command->request.methodLine, "SuppressCommandPort")) {
-        suppressCommandPort("SuppressCommandPort", true, true);
+        blockCommandPort("MANUAL");
```

Comment:
NAB we could make this a constant

Reply:
We could, but given that some of these are unspecified strings created by plugins, it seems less useful.